components/04-indicators/font-icon/font-icon.config.yml

```diff
@@ -0,0 +1,17 @@
+status: ready
+collated: true
+default: alert
+variants:
+  - name: alert
+    context:
+      icon: di di-warning
+  - name: info
+    # notes: Een font kan worden vergroot met modifier classes `.fa-2x` tot `.fa-6x`.
```

Comment:
Deze hulp classes wil ik ook vanuit `di` gaan aanbieden

Reply:
Aangepast. Worden wel weer overruled in de alert, omdat daar specifiek naar het ontwerp toe gestijld is.